olclient/openlibrary.py

```diff
@@ -106,6 +108,11 @@ def __init__(self, olid, **kwargs):
                 for kwarg in kwargs:
                     setattr(self, kwarg, kwargs[kwarg])
 
+            def json(self):
```

Comment:
I am going to make these `json()` methods return an object rather than a string to be like `request.Response.json()`